sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -179,6 +179,7 @@ case class Expand(
 trait GroupingAnalytics extends UnaryNode {
   self: Product =>
   def gid: AttributeReference
+  def groupings: Seq[AttributeReference]
```

Comment:
@chenghao-intel Do you mean calculate the `GROUPING` value by the `gid` and append it in the row during the `Expand` node?